packages/components/src/types/button-link-text.ts

```diff
@@ -0,0 +1,47 @@
+import { Generic } from '@a11y-ui/core';
+import { KoliBriButtonCallbacks, LinkTarget } from './button-link';
+import { Stringified } from './common';
+import { KoliBriIconProp } from './icon';
+import { Label } from './props/label';
+
+/**
+ * This types specifies the props of a link or button in navigations.
+ * We will support a mixin of link and button inside a navigation.
+ * Not all possible props of a link or button are relevant and supported.
```

Comment:
"**These** types **specify** the props of a button, link **or text**, as they are supported inside navigations."
Zweite Zeile steichen.